### ISSUE TITLE
chonology for todo

### ISSUE THREAD

**Issue by 9444055046** [madrecha/jamku#646]
dear sir
we find that the chronology is not enabled for todo and also for subtask.  can you please enable this

**Comment by adarshmadrecha** [comment]
Yes, currently, the chronology restriction is only applicable for Stages. 
Will add feature request to add  "chonology for todo".

**Comment by yogeshkudikala** [comment]
Dev Note -> We will have to think upon restricting users to change the sort nos
